config/environments/development.rb

```diff
@@ -73,4 +73,18 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.default_url_options = {host: "localhost:3000" }
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+  :address              => "smtp.gmail.com",
```

Comment:
```suggestion
  address: "smtp.gmail.com",
```
self check các line dưới hì